octo/run-short-tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+richgo test -short ./...
```

Comment:
```suggestion
richgo test -short ./...
richgo test ./... -run Unit
```

Apologies, but I'm still kind of stuck on this `-short` usage. That property isn't typically something you'd use in CI, I'd also prefer if we didn't have to have the `if` block you mentioned in all the tests to keep them from running.

I was looking through our codebase and for Paketo Java buildpacks, all of the unit tests already have the name `TestUnit` (I'd attribute this to copy & paste :)).

```
> find . -name "init_test.go" | xargs -n 1 grep -i -H 'func'
./dist-zip/distzip/init_test.go:func TestUnit(t *testing.T) {
./native-image/native/init_test.go:func TestUnit(t *testing.T) {
./occam/init_test.go:func TestUnitOccam(t *testing.T) {
./occam/packagers/init_test.go:func TestUnitPackagers(t *testing.T) {
./occam/matchers/init_test.go:func TestUnitMatchers(t *testing.T) {
./libbs/init_test.go:func TestUnit(t *testing.T) {
./google-stackdriver/stackdriver/init_test.go:func TestUnit(t *testing.T) {
./google-stackdriver/helper/init_test.go:func TestUnit(t *testing.T) {
./environment-variables/env/init_test.go:func TestUnit(t *testing.T) {
...
```

Given that, if we append `-run Unit` it should filter properly and only run the unit tests.